python-checks/src/test/resources/checks/booleanCheckNotInverted.py

```diff
@@ -0,0 +1,51 @@
+a = 1
+b = 2
+c = not (a == b)  # Noncompliant {{Use the opposite operator ("!=") instead.}}
+#   ^^^^^^^^^^^^
+cc = not (a != b)  # Noncompliant
+
+d = not (a > b)  # Noncompliant
+
+e = not (a <= b)  # Noncompliant
+
+f = float('nan')
+
+g = not (f > 2)  # Noncompliant
+
+h = not (f >= 2)  # Noncompliant
+
+i = not (((((a * 2)))))
+
+j = a and (not b)
+
+k = (not a) and (not b)
+
+l = not ((((((a > b))))))  # Noncompliant
+
+o = not a == b  # Noncompliant
+
+m = not a == b == 1
+
+n = not (not (a == b))  # Noncompliant
+
+p = not (1 == a == b) == 2 # Noncompliant
```

Comment:
* I'm not sure what this test is about
* It's not clear on which expression we expect an issue